### PR TITLE
Update Grid Layout

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Set value programmatically

Belongs to: #2714

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @erikjanwestendorp 😄 

I'll make sure it is reviewed as soon as possible.

**Review by sofietoft — COMMENTED** [review]
That sure is a lot of brackets to get through 😅 
But great sample!

Would it perhaps make sense to add a little section in the top of article on which kind of "settings" are used for the Grid in the sample? 🤔 
I'm guessing it's "just" the default one that Umbraco ships with that's used?

What do you think?

**Comment by erikjanwestendorp** [comment]
@sofietoft Sounds good!! 😄 I added an section 👍

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Add-Value-Programmatically.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+versionFrom: 8.0.0
+---
+
+# Add values programmatically
+
+## 	Configuration
+
+For this example, the default grid configuration has been used.
+
+![Grid configuration](images/Grid-configuration.jpg)
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Add-Value-Programmatically.md:11]:
```suggestion
![Grid configuration](Images/Grid-configuration.jpg)
```

**Review by sofietoft — COMMENTED** [review]
Thanks for applying the suggestions @erikjanwestendorp ! 😄 

I noticed that the image wasn't visible in the preview - might be because the images folder in this case is actually capitalized... 🤔 
I tried suggestion to update the path to the image - let's see if that does the trick, yeah? 🤞

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for the suggestion. I don't think it's because it's case sensitive but because the image isn't in the expected place yet. I think it will be fixed after merge. See the difference between:

https://github.com/umbraco/UmbracoDocs/pull/2966/files?short_path=73ba3a0#diff-73ba3a0c212e37d634ce61c5997644b22eff2469350375194634b0ae3e9e3cff

https://github.com/erikjanwestendorp/UmbracoDocs/blob/update-grid-layout-property-editor/Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Add-Value-Programmatically.md

**Review by sofietoft — APPROVED** [review]
Looks like that did the trick!

Thanks again for adding this article @erikjanwestendorp !
I'll get it merged in 💪

**Comment by sofietoft** [comment]
Ah. Merged before I saw your last comment there 🙈 

But of course it looks like you were right 😄 
The image (and the article) is live! 💪

**Pull request merged**